backend/corpora/common/entities/dataset.py

```diff
@@ -169,16 +169,29 @@ def get_assets(self):
         """
         return self.artifacts
 
+    @staticmethod
+    def parse_version(version):
```

Comment:
There's a nice Python package that does this instead of us writing our own custom code: https://pypi.org/project/semver/. I know it's a simple change, but it helps reduce the general surface area of this file, especially since this function is a general utility function, not something specific to the object that this class represents.

Reply:
I don't know how I feel about adding another dependency for a single usage. Also, I just realized that versions are lexicographically sorted so we should be able to just do `< "2.0.0"` and it should work.